{
	"id": "CVE-2020-29657",
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2020-29657",
				"description": "In JerryScript 2.3.0, there is an out-of-bounds read in main_print_unhandled_exception in the main-utils.c file.",
				"severity": [
					{
						"type": "cvss_v2",
						"source": "nvd@nist.gov",
						"cvss_v2": {
							"vector": "AV:N/AC:L/Au:N/C:P/I:N/A:P",
							"base_score": 6.4,
							"nvd_base_severity": "MEDIUM",
							"temporal_score": 6.4,
							"nvd_temporal_severity": "MEDIUM",
							"environmental_score": 6.4,
							"nvd_environmental_severity": "MEDIUM"
						}
					},
					{
						"type": "cvss_v31",
						"source": "nvd@nist.gov",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:H",
							"base_score": 9.1,
							"base_severity": "CRITICAL",
							"temporal_score": 9.1,
							"temporal_severity": "CRITICAL",
							"environmental_score": 9.1,
							"environmental_severity": "CRITICAL"
						}
					}
				],
				"cwe": [
					{
						"source": "nvd@nist.gov",
						"cwe": [
							"CWE-125"
						]
					}
				],
				"references": [
					{
						"source": "af854a3a-2127-422b-91ae-364da2661108",
						"url": "https://github.com/jerryscript-project/jerryscript/issues/4244"
					},
					{
						"source": "cve@mitre.org",
						"url": "https://github.com/jerryscript-project/jerryscript/issues/4244"
					},
					{
						"source": "nvd.nist.gov",
						"url": "https://nvd.nist.gov/vuln/detail/CVE-2020-29657"
					}
				],
				"published": "2020-12-09T09:15:13.137Z",
				"modified": "2024-11-21T05:24:22.207Z"
			},
			"segments": [
				{
					"ecosystem": "cpe"
				}
			]
		}
	],
	"detections": [
		{
			"ecosystem": "cpe",
			"conditions": [
				{
					"criteria": {
						"operator": "OR",
						"criterias": [
							{
								"operator": "OR",
								"criterias": [
									{
										"operator": "OR",
										"criterias": [
											{
												"operator": "OR",
												"criterions": [
													{
														"type": "version",
														"version": {
															"vulnerable": true,
															"fix_status": {
																"class": "unknown"
															},
															"package": {
																"cpe": "cpe:2.3:a:jerryscript:jerryscript:2.3.0:*:*:*:*:*:*:*"
															}
														}
													}
												]
											}
										]
									}
								]
							}
						]
					}
				}
			]
		}
	],
	"data_source": {
		"id": "nvd-api-cve",
		"raws": [
			"vuls-data-raw-nvd-api-cve/2020/CVE-2020-29657.json"
		]
	}
}
